docs/beaver-builder/troubleshooting/debugging/enable-beaver-builder-debug-mode.md

```diff
@@ -9,15 +9,18 @@ Builder Support team quickly get facts about your website's environment to
 help them troubleshoot. Debug mode can also help you find information to
 assist in your own troubleshooting efforts.
 
+For example, Debug Mode lets you access the any code you've added to the [Global CSS](../../styles/code/custom-css.md/#1-site-wide-rules-for-content-area-layouts) and [JavaScript](../../styles/code/custom-javascript.md) option on the Tools menu, so you can troubleshoot whether your custom code is causing a conflict with Beaver Builder.
```

Comment:
Fix this:

For example, Debug Mode lets you access `the` any code you've added to the